plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -0,0 +1,247 @@
+        result["entries"] = sessions_results
+        result["ret"] = True
+        return result
+
+    def set_NTPServer(self, mgr_attributes):
+        result = {}
+        setkey = mgr_attributes['mgr_attr_name']
+
+        nic_info = self.get_manager_ethernet_uri()
+        ethuri = nic_info["nic_addr"]
+
+        response = self.get_request(self.root_uri + ethuri)
+        if(response['ret'] is False):
+            return response
+        result['ret'] = True
+        data = response['data']
+
+        # enable_value = "false"
+
+        payload = {"DHCPv4": {
+            "UseNTPServers": ""
+        }}
+
+        if(data["DHCPv4"]["UseNTPServers"] is True):
+            payload["DHCPv4"]["UseNTPServers"] = False
+            resDHV4 = self.patch_request(self.root_uri + ethuri, payload)
+            if resDHV4['ret'] is False:
+                return resDHV4
+
+        payload = {"DHCPv6": {
+            "UseNTPServers": ""
+        }}
+
+        if(data["DHCPv6"]["UseNTPServers"] is True):
+            payload["DHCPv6"]["UseNTPServers"] = False
+            resDHV6 = self.patch_request(self.root_uri + ethuri, payload)
+            if resDHV6['ret'] is False:
+                return resDHV6
+
+        DTuri = self.manager_uri + "DateTime"
+
+        response = self.get_request(self.root_uri + DTuri)
+        if(response['ret'] is False):
+            return response
+
+        data = response['data']
+
+        NTPlist = data[setkey]
+        if(len(NTPlist) == 2):
+            NTPlist.pop(0)
+
+        NTPlist.append(mgr_attributes['mgr_attr_value'])
+
+        payload = {}
+        payload[setkey] = NTPlist
+
+        response1 = self.patch_request(self.root_uri + DTuri, payload)
+        if response1['ret'] is False:
+            return response1
+
+        return {'ret': True, 'changed': True, 'msg': "Modified %s" % mgr_attributes['mgr_attr_name']}
+
+    def setTimeZone(self, attr):
+        # result = {}
+        key = attr['mgr_attr_name']
+
+        uri = self.manager_uri + "DateTime/"
+        response = self.get_request(self.root_uri + uri)
+        if(response['ret'] is False):
+            return response
+
+        data = response["data"]
+
+        if key not in data:
+            return {'ret': False, 'msg': "Key %s not found" % key}
+
+        TimeZonelist = data["TimeZoneList"]
+        index = ""
+
+        for i in range(0, len(TimeZonelist)):
+            if(attr['mgr_attr_value'] in TimeZonelist[i]["Name"]):
+                index = TimeZonelist[i]["Index"]
+                break
+
+        payload = {key: {"Index": index}}
+        # toset = "{\"Index\":" + str(index) + "}"
+
+        # payload = {key: json.loads(toset)}
+
+        response = self.patch_request(self.root_uri + uri, payload)
+        if(response['ret'] is False):
+            return response
+
+        return {'ret': True, 'changed': True, 'msg': "Modified %s" % attr['mgr_attr_name']}
+
+    def set_DNSserver(self, attr):
+        # result = {}
```

Comment:
this looks like old code that is no longer used, please remove - it doesn't help the readability of the code